src/net/relay.rs

```diff
@@ -1512,6 +1511,11 @@ impl PeerNetwork {
         Ok(())
     }
 
+    // todo(ludo): open question - should nodes be advertising the fact that they just received an attachment?
+    pub fn advertize_attachments(&mut self) -> Result<(), net_error> {
```

Comment:
Please remove this method since it's a no-op.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/2795bc61271d427d7bbeb53c45c36edbacc99339